notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -229,3 +229,47 @@ sports [A group of friends were shown enjoying a leisurely game of croquet.]
 the game of tennis [Otis showed up at the house holding an old-timey, wooden framed tennis racket and was annoyed that his friend Charlie had forgotten about the game they had scheduled to play.]
 
 
+theamericanshortstory06
+=======================
+
+:: Title
+I'm a Fool
+
+:: Description
+A young horse trainer learns the hard way that telling a whopper of a lie is
+probably not the best way to win over the girl you fancy. It is based on the
+1923 Sherwood Anderson short story of the same name. 
+
+:: Date
+1977-04-05
+
+:: References
+https://www.imdb.com/title/tt0074662/
+
+:: Ratings
+4 <brent>
+4 <paul>
+
+:: Major Themes
+growing as a person [The story culminates with Andy realizing he was a fool for having tried to woo his love interest, Lucy, with lies of importance and wealth.]
+romantic love [Andy was smitten by Lucy, and Lucy was head over heels for the man who Andy feigned to be. Andy waxed poetic about how he wanted his first time to be with a princess.]
+mentor and protégé [Burt taught Andy the ins and outs of training race horses.]
+rite of passage into manhood [Andy went from being a young, bashful, and sexually inexperienced horse trainer to being a cigar-smoking, whiskey-drinking son of a gun, a man of the world and about town if you will, who was not above wooing girls at the horse races with lies of importance and wealth.]
```

Comment:
The comment sounds like https://www.themeontology.org/theme.php?name=coming%20of%20age but it is unclear what the "rite of passage" was

Reply:
examples of rites we've noticed: Vulcan warrior rite, killing a man, putting on "manly robes".

Reply:
Yes, in retrospect, I think it is reasonable to use **coming of age** as a major theme with the present justification for **rite of passage** as motivation, and move **rite of passage** to minor themes with a motivation to be written.

Reply:
I ended up deleting **rite of passage into manhood** on account of it being too tricky to motivate. I'll leave the pleasure of merging to you.